Commands/view_in_browser.py

```
@@ -0,0 +1,48 @@
+
+def reply_with_view_in_browser(self, message):
+    try:
+        github_site = config.GITHUB_SITE
+        if not github_site:
+            return
+
+        cmd = fetch_cmd_data(self, message)
+
+        if not check_cooldown(cmd.state, cmd.nick, cmd.cooldown):
+            return
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)